pdf-watermark/pdf-watermark.py

```diff
@@ -0,0 +1,33 @@
+from PyPDF2 import PdfFileWriter
+from PyPDF2 import PdfFileReader
+
+
+ori_pdf = PdfFileReader('inputfile.pdf')
+mark = PdfFileReader('watermark2.pdf')
+
+
+watermark = mark.getPage(0)
+original_pdf = ori_pdf.getPage(0)
+
+ori_page_count = ori_pdf.getNumPages()
+
+output = open('finalfile.pdf','wb')
```

Comment:
Change `finalfile.pdf` to `watermarked.pdf` for user convenience

Reply:
done